spec/structured-data.html

```diff
@@ -160,12 +203,8 @@ <h1>
             1. Return *"null"*.
           1. If Type(_value_) is BigInt, throw a *TypeError* exception.
           1. If Type(_value_) is Object and IsCallable(_value_) is *false*, then
-            1. <del>Let _isArray_ be ? IsArray(_value_).</del>
-            1. <ins>If ! IsTuple(_value_) is *true*, then</ins>
-              1. <ins>Let _isArrayLike_ be *true*.</ins>
-            1. <ins>Else,</ins>
-              1. <ins>Let _isArrayLike_ be ? IsArray(_value_).</ins>
-            1. If <del>_isArray_</del><ins>_isArrayLike_</ins> is *true*, return ? SerializeJSONArray(_state_, _value_).
+            1. Let <del>_isArray_</del><ins>_isArrayOrTuple_</ins> be ? <del>IsArray</del><ins>IsArrayOrTuple</ins>(_value_).</del>
```

Comment:
Seems like this shouldn't be indented under `If Type(_value_) is Object and IsCallable(_value_) is *false*, then...`? For primitive tuple values, `Type(_value_)` won't be `Object`.

Also, what about records?

Reply:
R&T are converted into object types a few lines earlier

> <ins>If Type(_value_) is Record or Type(_value_) is Tuple, set _value_ to ! ToObject(_value_).</ins>